tests/unit/layouts/ModelizerDrawLayout.spec.js

```diff
@@ -142,5 +142,25 @@ describe('Test page component: ModelizerDrawLayout', () => {
 
       expect(Notify.create).not.toHaveBeenCalled();
     });
+
+    it('should arrange a component position and redraw on click', async () => {
+      let awaitedArrangeComponentsPosition = false;
+      wrapper.vm.data.plugin = {
```

Comment:
Set mock before when we : `jest.mock('src/composables/PluginManager', () => ({`
just like :
```js
parse: jest.fn(),
draw: jest.fn(),
```

Reply:
Then how could I check that plugin.arrangeComponentsPosition() has been properly awaited ?
Or maybe I don't need to check this ?

Reply:
done